queries.sql

```diff
@@ -0,0 +1,8 @@
+SELECT * FROM animals WHERE name LIKE '%mon';
+SELECT name FROM animals WHERE date_of_birth BETWEEN '01-01-2016' AND '12-31-2019';
+SELECT name FROM animals WHERE neutered=true AND escape_attempts<=3;
```

Comment:
👍 Great job! This SQL query effectively retrieves the names of animals from the "animals" table that are both neutered and have either less than or equal to 3 escape attempts. By using the `WHERE` clause with the conditions `neutered=true` and `escape_attempts<=3`, the query ensures that it includes animals that are neutered and those with up to 3 escape attempts, providing a concise summary of the desired information, which encompasses animals with 0, 1, 2, or 3 escape attempts. 

- [ ] However, please consider the project requirement, which states: "List the name of all animals that are neutered and have less than 3 escape attempts." To align with this requirement, you should adjust your query to include animals with fewer than 3 escape attempts. Additionally, if needed, update the Output Screenshot in the Pull Request Description to reflect the correct results. Keep up the excellent work! 👏🐾🐕

Reply:
Thanks for the feedback